_mobile/ios.md

```diff
@@ -194,6 +193,37 @@ In the build settings, search for **other linker flags**.  Add a custom linker f
 
 Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here https://pytorch.org/cppdocs/. To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
 
+
+### Custom build
+
+Starting from 1.4.0, PyTorch supports custom build. You can now build the PyTorch library that only contains the operators needed by your model. To do that, follow the steps below
+
+1. Verify your PyTorch version is 1.4.0 or above. You can do that by checking the value of `torch.__version__`.
+2. To dump the operators in your model, we need to run a few lines of Python code
+
+    ```python
+    import torch, yaml
+    model = torch.jit.load("example.pt")
+    ops = torch.jit.export_opnames(m)
+    f = open('example.yaml', 'w')
+    yaml.dump(ops, f)
+    ```
+The `export_opnames` API is available in `torch.jit` module, and we need to save the result to a yaml file.
+
+3. Run the iOS build script locally and pass in the yaml file generated from last step
+
+    ```
+    SELECTED_OP_LIST=example.yaml BUILD_PYTORCH_MOBILE=1 IOS_ARCH=arm64 ./scripts/build_ios.sh
```

Comment:
Suggestion to specify using the SELECTED_OP_LIST environment variable and elaborate on the additional arguments.

Also consider adding link to docs for this environment variable and its arguments

Suggested rephrase - 
"To run the iOS build script locally with the prepared yaml list of operators, pass in the yaml file generate from the last step into the environment variable SELECTED_OP_LIST. Also in the arguments, specify X, Y, Z (X, Y, Z being the arguments  BUILD_PYTORCH_MOBILE, IOS_ARCH, ./scripts/build_ios.sh)."